### DISCUSSION TITLE
getQueryResultWithPagination bookmark value

### DISCUSSION THREAD

**Discussion by victoryeo** [hyperledger/fabric#2694] [Q&A]
This chaincode function, getQueryResultWithPagination(query, pageSize, bookmark) , bookmark value is like this:
"g1AAAABWeJzLYWBgYMpgSmHgKy5JLCrJTq2MT8lPzkzJBYoLGwKBaQ2INNQ1hACQUg6YUlyKsgCcDxS3"

If we want to read the data at 10,001 location, is there a way to skip the first 10,000 data ?
Something like this: 
getQueryResultWithPagination(query, 10000, "1") 

Is it possible to support this feature when using couchDB
In couchDB, there are limit and skip that can be passed to query iterator.

**Comment by HarshMultani** [comment]
Can anyone point me to an example code that uses this function getQueryResultWithPagination() and is able to retrieve the records through pagination and sorting
